sources/logdog/.gitignore

```
@@ -0,0 +1,2 @@
+# ignore the symlink generated by build.rs
+conf/current/
```

```suggestion
/conf/current/
```
nit: It shouldn't actually make a difference here, but the starting slash at least makes clear to future readers that this directory lives at the top level.

---

[aa7d592](https://github.com/bottlerocket-os/bottlerocket/compare/60e5d3c8e65c58170e8f72d944a80be11ea098e0..aa7d59202dad5161a9dc8aff3995f1fbe0d4c706)